sqlite/src/main/java/org/jdbi/v3/sqlite3/SQLitePlugin.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.sqlite3;
+
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.spi.JdbiPlugin;
+
+import java.net.URL;
+
+/**
+ * Jdbc plugin for SQLite
```

Comment:
This is a `jdbi` plugin :)
Also the javadoc is missing the required `.` termination.

Reply:
lol...the names got mixed up in my head.

Reply:
I don't know why, they're totally different :trollface:

Reply:
😅